README.md

```diff
@@ -1,6 +1,32 @@
-# coursebook
+# Founders and Coders Curriculum
 
-The new coursebook
+Welcome to our web development curriculum. This is the material we use to teach beginners to be developers in our free bootcamp.
```

Comment:
Could we avoid using the word 'teach'? I'm not sure it reflects the learning environment we create. I could suggest: "This the the material which forms the basis of our tuition-free, peer-led web development programme."

Reply:
Good catch, thanks